pages/static/hta1.html

```diff
@@ -0,0 +1,130 @@
+---
+page: hta1
+navText: Human Tumor Atlas Pilot Project (HTAPP)
+---
+
+<h1>Human Tumor Atlas Pilot</h1>
+<div style="display: flex; justify-content: space-evenly">
+    <img src="/Broad-Inst-Logo.webp" style="width: 30%; height: 100%" />
+    <img src="/DFCI-Logo-with-Flag.webp" style="width: 30%; height: 100%" />
+    <img src="/HMS_Logo.webp" style="width: 30%; height: 100%" />
+</div>
+<br />
+<a href="https://humantumoratlas.org/htapp-webinar-series/">
```

Comment:
This webinar link button has no style yet, but good to make some tweaks to fits other objects